src/ngraph/op/experimental/range.hpp

```diff
@@ -44,13 +44,25 @@ namespace ngraph
                 ///             element type, and same element type as `start` and `stop`.
                 Range(const Output<Node>& start,
                       const Output<Node>& stop,
-                      const Output<Node>& step);
+                      const Output<Node>& step,
+                      int64_t max_output_length = -1);
```

Comment:
Why not define a constant called something like `UNBOUNDED` = -1 and then use it here and on line 65 below? That would keep them in sync and remove the need for documenting what -1 means.